tests/pr_test.go

```diff
@@ -54,6 +55,37 @@ func TestRunBasicExample(t *testing.T) {
 	assert.NotNil(t, output, "Expected some output")
 }
 
+func TestRunAdvanceExample(t *testing.T) {
+	t.Parallel()
+
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		Prefix:  "advanced-key-protect",
+		TarIncludePatterns: []string{
+			advancedExampleTerraformDir + "/*.tf",
```

Comment:
please add back the root level - the example calls the root level module so both are needed in the tar file that gets sent to schematics
```
TarIncludePatterns: []string{
			"*.tf",
			advancedExampleTerraformDir + "/*.tf",
		},
```

Reply:
@ocofaigh done.